cnd/src/network/comit_ln.rs

```diff
@@ -211,6 +228,10 @@ impl ComitLN {
     }
 }
 
+#[derive(thiserror::Error, Clone, Copy, Debug)]
+#[error("swap already exists")]
+pub struct SwapExists {}
```

Comment:
Nit: If you decide to not include an data, you can do this (saves you 3 characters when instantiating it because you can just say `anyhow::bail!(SwapExists)`:
```suggestion
pub struct SwapExists;
```

If we decide to log this error instead of sending it to the user, it might be helpful to include the digest in the log message so it shows up in the log file.

Reply:
Cool thanks.